users/serializers.py

```diff
@@ -245,7 +245,7 @@ class UserSerializer(serializers.ModelSerializer):
 
     def validate_email(self, value):
         """Empty validation function, but this is required for WriteableSerializerMethodField"""
-        return {"email": value}
+        return {"email": value.strip().lower()}
```

Comment:
Do we need to lower the email here too?

Reply:
I added this check here if a user updates his/her email. It'll always be saved in lowercase. At the time of registration and on updating his/her account info

Reply:
I've removed this `.strip().lower()` check from the pipeline so now it is definitely required here